superset-frontend/src/components/ErrorMessage/ErrorAlert.tsx

```diff
@@ -70,7 +70,7 @@ const ErrorModal = styled(Modal)<{ level: ErrorLevel }>`
   color: ${({ level, theme }) => theme.colors[level].dark2};
   overflow-wrap: break-word;
 
-  .ant-modal-header {
+  .antd5-modal-header {
     background-color: ${({ level, theme }) => theme.colors[level].light2};
```

Comment:
Can this be centralized / removed / or moved to theme level?

Reply:
This a customized modal and the color of the header depends on the level of error meaning `info` , `warning` , `error`, I think this should stay as it is